internal/git/provider/providerfakes/fake_provider.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	"github.com/go-logr/logr"
-	"github.com/weaveworks/tf-controller/internal/git/provider"
+	"github.com/flux-iac/tofu-controller/internal/git/provider"
```

Comment:
This file is auto-generated, where it sorts the import every time.

It seems to be re-generated or you could re-arrange these 2 lines to be:

```
"github.com/flux-iac/tofu-controller/internal/git/provider"
"github.com/go-logr/logr"
```

Reply:
@akselleirv sorry for another ping.
there's a small glitch left because this fake provider is auto-generated and formatted.

Reply:
Thanks! Fixed in https://github.com/flux-iac/tofu-controller/pull/1207/commits/b528d396aa3980491f29a5c798fe5f579d7fbe92